README.md

```diff
@@ -14,9 +14,9 @@ This repository is [Maintained](https://github.com/packethost/standards/blob/mas
 * [Requirements](#requirements)
 * [Supported Platforms](#supported-platforms)
 * [Installation](#installation)
-  * [Linux](#linux)
```

Comment:
Interesting. We still have these sections but they are behind `<details>` blocks. I do prefer the sections that you've highlighted.